platform/integrations/slack_integration/usage.mdx

```diff
@@ -0,0 +1,104 @@
+---
+id: usage
+title: Usage
+sidebar_label: Usage
+description: Learn how to use commands, receive notifications about Tolgee project updates in Slack, customize the notifications, and more.
+---
+
+import { FigmaScreenshotWrapper } from '../../shared/_FigmaScreenshotWrapper';
+
+# Usage
+
+This guide will help you subscribe to project updates, manage your notifications, and more. Whether you're a project manager, developer, or translator,
+ Tolgee's Slack integration offers a seamless way to streamline your workflow and collaborate more efficiently with your team. :rocket:
+
+## Subscribe to a Project
+
+Subscribe to a project to receive notifications for all events and languages in the project.
+
+``
+/tolgee subscribe $project_id
```

Comment:
We should be consistent in how we document the syntax. According to standards positional arguments should be wrapped in `<>` So this should be. 
```suggestion
/tolgee subscribe <project_id>
```

Reply:
Also, there should be some information how to retrieve project ID. There is currently doc page about this missing, can you please add it?